server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraBlobStoreDependenciesModule.java

```diff
@@ -44,10 +42,6 @@ protected void configure() {
             .annotatedWith(Names.named(CassandraDumbBlobStore.DEFAULT_BUCKET))
             .toInstance(BucketName.DEFAULT);
 
-        bind(BlobStore.class)
```

Comment:
why did you split that module?

Reply:
To avoid this to be shipped as a dafault for distributed james and avoid a module override.